src/main/java/net/openhft/chronicle/queue/impl/single/RollCycleEncodeSequence.java

```diff
@@ -21,9 +21,8 @@
 import net.openhft.chronicle.core.Maths;
 import net.openhft.chronicle.core.values.LongValue;
 import net.openhft.chronicle.core.values.TwoLongValue;
-import net.openhft.chronicle.wire.Sequence;
```

Comment:
Is it necessary? It was more concise.

Reply:
Unfortunately yes. You can't put a `@SuppressWarning("deprecation")` on an import, so you need to use the fully qualified name. 
BTW This PR un-deprecates it anyway... https://github.com/OpenHFT/Chronicle-Wire/pull/871